packages/hosted-assets/Kaizen/HostedAssets/Image.elm

```diff
@@ -0,0 +1,70 @@
+module Kaizen.HostedAssets.Image exposing (Config, Role(..), attributes, default, image)
+
+import Html exposing (Html, img)
+import Html.Attributes exposing (src)
+import Html.Attributes.Aria as Aria exposing (ariaHidden, ariaLabelledby)
+import Kaizen.HostedAssets exposing (assetUrl)
+
+
+type Role
+    = Presentation
```

Comment:
This just supports presentation for now (completely hidden from screen readers), but other types where you want a description to be read to a screen reader could be added.